generator/generator-opc-generator/generators/app/templates/README.md

```diff
@@ -0,0 +1,41 @@
+# This is <%= componentName %> 👋
+
+<%= componentDescription %>
```

Comment:
@hybridx u need to update this README to include specific commands to run, test, build the specific component.

Reply:
The commands present work for the component